spark/src/main/scala/geotrellis/spark/io/s3/S3LayerCopier.scala

```diff
@@ -0,0 +1,69 @@
+package geotrellis.spark.io.s3
+
+import com.amazonaws.services.s3.model.ObjectListing
+import geotrellis.spark.io.index.KeyIndex
+import geotrellis.spark.{KeyBounds, LayerId}
+import geotrellis.spark.io._
+import geotrellis.spark.io.json._
+import org.apache.avro.Schema
+import spray.json.JsonFormat
+import scala.annotation.tailrec
+import scala.collection.JavaConversions._
+import scala.reflect.ClassTag
```

Comment:
Please watch the import organization. Would look better like this:

```scala
import geotrellis.spark.io.index.KeyIndex
import geotrellis.spark.{KeyBounds, LayerId}
import geotrellis.spark.io._
import geotrellis.spark.io.json._

import com.amazonaws.services.s3.model.ObjectListing
import org.apache.avro.Schema
import spray.json.JsonFormat
import scala.annotation.tailrec
import scala.collection.JavaConversions._
import scala.reflect.ClassTag
```

geotrellis imports first, then external library imports, grouped by library. You may or may not want to separate the different libraries with a blank space. Java and Scala imports go last.